src/main/java/net/sacredlabyrinth/phaed/simpleclans/Clan.java

```diff
@@ -1549,11 +1549,13 @@ public boolean isWarring(Clan clan) {
     /**
      * Add a clan to be at war with
      */
-    public void addWarringClan(Clan clan) {
```

Comment:
Breaking change.
Re-add the method as:
```java
public void addWarringClan(Clan target) {
    addWarring(null, target);
}
```

Reply:
Do I need to use it in ProtectionManager?
From:
```java
        requestClan.addWarringClan(requester, targetClan);
        targetClan.addWarringClan(requester, requestClan);
```
To:
```java
        requestClan.addWarringClan(targetClan);
        targetClan.addWarringClan(requestClan);
 ```

Reply:
Nope.